### PR TITLE
2.5: Update quickstart repo file paths

### PR DESCRIPTION
**Pull request by btat**
Fixes https://github.com/rancher/rancher-docs/issues/380

### CODE REVIEW

File: versioned_docs/version-2.5/getting-started/quick-start-guides/deploy-rancher-manager/aws.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The AWS module just creates an EC2 KeyPair, an EC2 SecurityGroup and an EC2 inst
 
 1. Clone [Rancher Quickstart](https://github.com/rancher/quickstart) to a folder using `git clone https://github.com/rancher/quickstart`.
 
-2. Go into the AWS folder containing the terraform files by executing `cd quickstart/aws`.
+2. Go into the AWS folder containing the terraform files by executing `cd quickstart/rancher/aws`.
```
Comment (martyav) [inline versioned_docs/version-2.5/getting-started/quick-start-guides/deploy-rancher-manager/aws.md:40]: Suggest "Terraform" (capitalized) over "terraform"

File: versioned_docs/version-2.5/getting-started/quick-start-guides/deploy-rancher-manager/azure.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The following steps will quickly deploy a Rancher server on Azure in a single-no
 
 1. Clone [Rancher Quickstart](https://github.com/rancher/quickstart) to a folder using `git clone https://github.com/rancher/quickstart`.
 
-2. Go into the Azure folder containing the terraform files by executing `cd quickstart/azure`.
+2. Go into the Azure folder containing the terraform files by executing `cd quickstart/rancher/azure`.
```
Comment (martyav) [inline versioned_docs/version-2.5/getting-started/quick-start-guides/deploy-rancher-manager/azure.md:26]: Similar to above, suggest capitalizing Terraform.

File: versioned_docs/version-2.5/getting-started/quick-start-guides/deploy-rancher-manager/digitalocean.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ The following steps will quickly deploy a Rancher server on DigitalOcean in a si
 
 1. Clone [Rancher Quickstart](https://github.com/rancher/quickstart) to a folder using `git clone https://github.com/rancher/quickstart`.
 
-2. Go into the DigitalOcean folder containing the terraform files by executing `cd quickstart/do`.
+2. Go into the DigitalOcean folder containing the terraform files by executing `cd quickstart/rancher/do`.
```
Comment (martyav) [inline versioned_docs/version-2.5/getting-started/quick-start-guides/deploy-rancher-manager/digitalocean.md:23]: Suggest capitalizing "terraform"

File: versioned_docs/version-2.5/getting-started/quick-start-guides/deploy-rancher-manager/gcp.md [hunk 4]
```diff
@@ -21,7 +21,7 @@ The following steps will quickly deploy a Rancher server on GCP in a single-node
 
 1. Clone [Rancher Quickstart](https://github.com/rancher/quickstart) to a folder using `git clone https://github.com/rancher/quickstart`.
 
-2. Go into the GCP folder containing the terraform files by executing `cd quickstart/gcp`.
+2. Go into the GCP folder containing the terraform files by executing `cd quickstart/rancher/gcp`.
```
Comment (martyav) [inline versioned_docs/version-2.5/getting-started/quick-start-guides/deploy-rancher-manager/gcp.md:24]: Suggest capitalizing "terraform"

**Review by martyav — COMMENTED** [review]
I left several suggestions to capitalize "terraform" since we capitalize other tool names in similar contexts (example: "Docker files", not "docker files").

File: versioned_docs/version-2.5/getting-started/quick-start-guides/deploy-rancher-manager/aws.md (hunk 1 above)
Comment (martyav) [inline versioned_docs/version-2.5/getting-started/quick-start-guides/deploy-rancher-manager/aws.md:40]:
```suggestion
2. Go into the AWS folder containing the Terraform files by executing `cd quickstart/rancher/aws`.
```

Comment (btat) [reply versioned_docs/version-2.5/getting-started/quick-start-guides/deploy-rancher-manager/aws.md]: Updated and applied to other versions of similar pages as well.

**Review by martyav — APPROVED** [review]

**Pull request merged**